examples/opf/tools/sp_plotter.py

```diff
@@ -226,16 +226,15 @@ def testSP():
   while numRecords < 3:
     
     # Setup a SP
-    sp = FDRCSpatial2.FDRCSpatial2(
-           coincidencesShape=(2048, 1),
-           inputShape = (1, elemSize),
-           inputBorder = elemSize/2-1,
```

Comment:
:+1:  I came across some documentation on this one a while ago, as @subutai explained, it was used for image processing, where 2D topology and "lonely columns at the borders" played part. It is not used in newer SP implementations.